website/docs/docs/build/incremental-microbatch.md

```diff
@@ -25,7 +25,8 @@ Incremental models in dbt are a [materialization](/docs/build/materializations)
 Microbatch is an incremental strategy designed for large time-series datasets:
 - It relies solely on a time column ([`event_time`](/reference/resource-configs/event-time)) to define time-based ranges for filtering. Set the `event_time` column for your microbatch model and its direct parents (upstream models). Note, this is different to `partition_by`, which groups rows into partitions.
 - It complements, rather than replaces, existing incremental strategies by focusing on efficiency and simplicity in batch processing.
-- Unlike traditional incremental strategies, microbatch doesn't require implementing complex conditional logic for [backfilling](#backfills). 
+- Unlike traditional incremental strategies, microbatch doesn't require implementing complex conditional logic for [backfilling](#backfills), allows you to [reprocess failed batches](/docs/build/incremental-microbatch#retry), and auto-detects [parallel batch execution](#parallel-batch-execution).
```

Comment:
Removing the double negative to make it easier to read (unlike / doesn't allow):
```suggestion
- Unlike traditional incremental strategies, microbatch allows you to [reprocess failed batches](/docs/build/incremental-microbatch#retry), auto-detect [parallel batch execution](#parallel-batch-execution), and eliminates the need to implement complex conditional logic for [backfilling](#backfills).
```